frontend/src/routes/Infrastructure/Clusters/ClusterSets/ClusterSetDetails/ClusterSetInstallSubmariner/InstallSubmarinerForm.tsx

```diff
@@ -168,6 +173,10 @@ export function InstallSubmarinerForm(props: { availableClusters: Cluster[] }) {
     const [withoutSubmarinerConfigClusters, setWithoutSubmarinerConfigClusters] = useState<Cluster[]>([])
     const [fetchSecrets, setFetchSecrets] = useState<boolean>(true)
 
+    const [globalnetEnabled, setGlobalnetEnabled] = useState<boolean>(false)
+    const [isGlobalnetDisabled, setGlobalnetDisabled] = useState<boolean>(true)
```

Comment:
These are so similar in name. Instead of `isGlobalnetDisabled`/`setGlobalnetDisabled` can we call it something like `isGlobalnetAlreadyConfigured`?

Reply:
ok